state/resources_state_resource.go

```diff
@@ -21,6 +21,8 @@ import (
 	"github.com/juju/juju/resource"
 )
 
+var rLogger = logger.Child("resource")
```

Comment:
drive by to facilitate debugging

Reply:
I wonder if you could leave it, seems useful!